src/components/Person/Person.jsx

```diff
@@ -1 +1,32 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+  const nameInfo = `My name is ${name}`;
```

Comment:
In such case you can put this text directly into the tag. In general you want to assign something to a variable if you want to give it a name so it's more clear what it does. In this case it's self explanatory